generators/kubernetes/templates/db/postgresql.yml.ejs

```diff
@@ -86,7 +86,8 @@ spec:
         - containerPort: 5432
         volumeMounts:
         - name: data
-          mountPath: /var/lib/postgresql/
+          mountPath: /var/lib/postgresql/data
+          subMount: postgres
```

Comment:
should this be `subPath`?

Reply:
oh yeah, you're right...that was a brain error xD

I'm fixing this asap